vmdb/app/models/host.rb

```diff
@@ -1244,15 +1244,21 @@ def detect_discovered_os(ost)
   def detect_discovered_hypervisor(ost, ipaddr)
     log_header = 'MIQ(Host.detect_discovered_hypervisor)'
     find_method = :find_by_ipaddress
+
+    # TODO: break this apart, we probably want to only do the namelookup for hyperv, esx
+    hostname = Socket.getaddrinfo(ipaddr, nil)[0][2]
```

Comment:
@tenderlove Is there a timeout value you can give to `Resolv`?  If not, maybe we have to wrap in a timeout block?  In the past reverse DNS has bit us when there is no reverse, or it fails, since the default timeout is WAY too long.